### PR TITLE
Improved Sidebar UI by making it more minimal looking

### PR DESCRIPTION
**Pull request by gvirus21**
# Description

This edit improves the look of the sidebar of your portfolio by giving it a more minimal look.

I removed the underline on the selected menu items to make it a little aesthetically pleasing and more minimal look in general.

## Type of change

- UI updated

### CODE REVIEW

File: src/components/NavBar/NavBar.css [hunk 1]
```diff
@@ -1,90 +1,83 @@
...
 .bm-menu-wrap {
-    position: fixed;
-    height: 100vh;
-    overflow-x: hidden;
-    box-shadow: 0px 4px 31px rgba(202, 59, 59, 0.15);
+  position: fixed;
+  height: 100vh;
+  overflow-x: hidden;
+  box-shadow: 0px 4px 31px rgba(202, 59, 59, 0.15);
+  background-color: red;
```
Comment (bislara) [inline src/components/NavBar/NavBar.css:37]: This red color doesn't look good. also, is this required to add?

**Review by bislara — COMMENTED** [review]
Why is there so many changes in package-lock.json? Also, This is fine. Could you please add some hover effect? That will look more better I feel. What say?

File: src/components/NavBar/NavBar.css (hunk 1 above)
Comment (gvirus21) [reply src/components/NavBar/NavBar.css:37]: yes i was just playing around with code for a min, I forgot to delete that background color: red statement.

**Comment by gvirus21** [comment]
> Why are there so many changes in package-lock.json? Also, This is fine. Could you please add some hover effect? That will look better I feel. What say?

yes, I can definitely add the hover effect.

**Pull request closed without merging**